src/app/components/MessageBanner/index.test.tsx

```diff
@@ -1,97 +1,119 @@
+        'H2',
+      );
+    });
 
-  it('should display the banner subtext correctly as a Paragraph', () => {
-    render(
-      <MessageBanner
-        summaries={summaries}
-        title={kyrgyzBanner.title}
-        position={kyrgyzBanner.position}
-      />,
-    );
-    expect(screen.getByText(summary.description).nodeName).toBe('P');
-  });
+    it('should display the banner subtext correctly as a Paragraph', () => {
+      render(
+        <MessageBanner
+          summary={summary}
+          title={kyrgyzMessageBannerOnePromo.title}
+          position={kyrgyzMessageBannerOnePromo.position}
+        />,
+      );
+      expect(screen.getByText(summary.description).nodeName).toBe('P');
+    });
 
-  it('should display link text correctly as an Anchor', () => {
-    render(
-      <MessageBanner
-        summaries={summaries}
-        title={kyrgyzBanner.title}
-        position={kyrgyzBanner.position}
-      />,
-    );
-    const ctaLink = screen.getByRole('link');
-    expect(ctaLink.getAttribute('href')).toEqual(summary.link);
-    expect(ctaLink.textContent).toEqual(summary.title);
-  });
+    it('should display link text correctly as an Anchor', () => {
+      render(
+        <MessageBanner
+          summary={summary}
+          title={kyrgyzMessageBannerOnePromo.title}
+          position={kyrgyzMessageBannerOnePromo.position}
+        />,
+      );
+      const ctaLink = screen.getByRole('link');
+      expect(ctaLink.getAttribute('href')).toEqual(summary.link);
+      expect(ctaLink.textContent).toEqual(summary.title);
+    });
+
+    it('should render an image with the correct image src', () => {
+      render(
+        <MessageBanner
+          summary={summary}
+          title={kyrgyzMessageBannerOnePromo.title}
+          position={kyrgyzMessageBannerOnePromo.position}
+        />,
+      );
+      const image = screen.getByAltText('');
+      expect(image.getAttribute('src')).toEqual(
+        summary.imageUrl.replace('{width}', 'raw'),
+      );
+    });
 
-  it('should render an image with the correct image src', () => {
-    render(
-      <MessageBanner
-        summaries={summaries}
-        title={kyrgyzBanner.title}
-        position={kyrgyzBanner.position}
-      />,
-    );
-    const image = screen.getByAltText('');
-    expect(image.getAttribute('src')).toEqual(
-      summary.imageUrl.replace('{width}', 'raw'),
-    );
+    it('should have an image with an empty alt text', () => {
+      render(
+        <MessageBanner
+          summary={summary}
+          title={kyrgyzMessageBannerOnePromo.title}
+          position={kyrgyzMessageBannerOnePromo.position}
+        />,
+      );
+      const image = screen.getByAltText('');
+      expect(image).toBeInTheDocument();
+    });
   });
+  describe('Two summaries', () => {
+    const { summaries } = kyrgyzMessageBannerTwoPromos;
+    it('should only render one banner per curation', () => {
```

Comment:
I know it's ironic but

...Karina pls sleep 🤣

Reply:
So thinking more about this test - we have a very similar ones in https://github.com/bbc/simorgh/blob/latest/src/app/pages/TopicPage/index.test.jsx#L167-L185 or https://github.com/bbc/simorgh/blob/latest/src/app/pages/TopicPage/index.test.jsx#L187-L198, and I am wondering if we could do something like this:

```
/**
 ** Find curation from the kyrgyzTopicWithMessageBanners with visual style = BANNER 
 ** and visual prominence = NORMAL AND summaries array length greater than 1   
 ** 
 ** Query ALL by role based on the curation title (because getByRole will only get the first one if it exists)
 ** Ensure the length of the array is 1
 **/
```

Reply:
Thanks for sorting this @louisearchibald 🧪